src/GraphPicker.jsx

```diff
@@ -40,27 +40,27 @@ export default class GraphPicker extends React.Component {
         const me = this;
         const graphs = [
             [8, 'ADAS.png', false],
-            [9, 'linear_demand_supply.png', true],
+            [9, 'linear_demand_supply_area.png', true],
```

Comment:
This is the kind of change that could definitely be merged on a separate branch, as it's independent of everything else. It's no big deal - we can merge it here. But having a separate PR for these changes would speed this fix going out, because of all the other stuff to review in this PR.